src/components/Sidebar/components/NavItem/utils.js

```diff
@@ -25,4 +27,17 @@ const hasSelectedChild = children => {
   return false;
 };
 
-export default hasSelectedChild;
+const disabledIcon = <Lock />;
```

Comment:
If you move this variable into the `getIcon` function, it could be tree-shaken when only `hasSelectedChild` is imported — although I don't know if that's ever the case.

Reply:
It could be the case! Nicely spotted 🙂